lib/processors/bundlers/moduleBundler.js

```diff
@@ -32,12 +32,18 @@ const log = require("@ui5/logger").getLogger("builder:processors:bundlers:module
  *		modules, a jQuery.sap.require will be created. In case the ui5loader is available, 'sap.ui.requireSync' is
  *		used instead.
  *	</li>
+ *	<li>
+ *		<code>bundleInfo</code>: A 'bundleInfo' section describes the content of another named bundle. This information
+ *		is transformed into a ui5loader-"bundlesUI5" configuration.
+ *		At runtime, if a module is known to be contained in a bundle, the loader will require that bundle before
+ *		the module itself is required. This requires UI5 version 1.74.0 or higher.
```

Comment:
It also requires that the ui5loader is available (which should be the case with 1.74 or higher - _when_ there's a dependency to sap.ui.core :-) )

Reply:
Done